static/css/dashboard.css

```diff
@@ -33,6 +33,11 @@ body, .containers-dashboard {
   --pf-l-stack--m-gutter--MarginBottom: var(--dashboard-gutter);
 }
 
+.monitoring-portal-link {
+  line-height: 1.5;
+  font-weight: 600;
+  font-size: 16px;
```

Comment:
Font size for link text "monitoring portal" is bigger than default dashboard font size, It is 16px instead of 15px.
Why do you think it's necessary to enlarge it?

Reply:
It is the same link style as is used on the welcome page.  I will adjust the values to use the PF CSS variables to be page consistent.